### PR TITLE
Add sample screens and navigation 

### PR DESCRIPTION
**Pull request by Lobynya**
Add sample screens and navigation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,40 +2,107 @@
 
 Sample application for https://github.com/cashapp/redwood
 
-![android-counter](https://user-images.githubusercontent.com/5010169/212373350-0e5b50ad-9775-43c1-92dd-cbfde7e0a1fc.gif)
```
Comment (Alex009) [inline README.md:5]: gif clipped - not show anything after login input

File: androidApp/build.gradle.kts [hunk 2]
```diff
@@ -41,7 +41,10 @@ dependencies {
     implementation("androidx.activity:activity-compose:1.6.1")
 
     implementation(project(":shared"))
-
     implementation("app.cash.redwood:redwood-layout-composeui:0.1.0")
+    implementation("app.cash.redwood:redwood-treehouse-lazylayout-api:0.1.0")
+    implementation("app.cash.redwood:redwood-treehouse:0.1.0")
```
Comment (Alex009) [inline androidApp/build.gradle.kts]: unused deps - remove

File: androidApp/src/main/res/drawable/empty.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Alex009) [inline androidApp/src/main/res/drawable/empty.xml]: unused?

File: build.gradle.kts [hunk 4]
```diff
@@ -8,6 +8,17 @@ plugins {
...
+
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath("dev.icerock.moko:resources-generator:0.20.1")
+    }
 }
```
Comment (Alex009) [inline build.gradle.kts]: duplicated logic - remove

File: schema/compose/build.gradle.kts [hunk 5]
```diff
@@ -15,6 +15,7 @@ kotlin {
     sourceSets {
         commonMain {
             dependencies {
+                api("app.cash.redwood:redwood-layout-compose:0.1.0")
```
Comment (Alex009) [inline schema/compose/build.gradle.kts]: duplicate

File: schema/build.gradle.kts [hunk 6]
```diff
@@ -6,5 +6,6 @@ dependencies {
     api("app.cash.redwood:redwood-schema:0.1.0")
     api("app.cash.redwood:redwood-layout-schema:0.1.0")
     api("dev.icerock.moko:resources:0.20.1")
+    //api("app.cash.redwood:redwood-treehouse-lazylayout-schema:0.1.0")
```
Comment (Alex009) [inline schema/build.gradle.kts]: remove

File: shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+                                                Icon(
+                                                    painter = painterResource(id = R.drawable.arrow_left),
+                                                    contentDescription = null
+                                                )
+                                            }
+                                        }
+                                        Text(
+                                            modifier = Modifier.align(Alignment.Center),
+                                            text = screenSettings.text.toString(LocalContext.current)
+                                        )
```
Comment (Alex009) [inline shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt]: why not used solution default? TopAppBar navigationIcon, title

File: shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+                                        it
+                                    )
+                                if (accountType != null) {
+                                    argsMap[it] = accountType
+                                }
+                            }
+
+                            val tmp = entry.arguments?.toString()?.substringAfter("dat=")
+                                ?.substringBefore('}')
+                            val params = tmp?.substringAfter('?')
```
Comment (Alex009) [inline shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt]: `dat`? what is it?

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: androidApp/build.gradle.kts (hunk 2 above)
Comment (Lobynya) [reply androidApp/build.gradle.kts]: fixed

File: androidApp/src/main/res/drawable/empty.xml (hunk 3 above)
Comment (Lobynya) [reply androidApp/src/main/res/drawable/empty.xml]: fixed

File: build.gradle.kts (hunk 4 above)
Comment (Lobynya) [reply build.gradle.kts]: fixed

File: schema/compose/build.gradle.kts (hunk 5 above)
Comment (Lobynya) [reply schema/compose/build.gradle.kts]: fixed

File: schema/build.gradle.kts (hunk 6 above)
Comment (Lobynya) [reply schema/build.gradle.kts]: fixed

File: shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt (hunk 7 above)
Comment (Lobynya) [reply shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt]: fixed

File: shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt (hunk 8 above)
Comment (Lobynya) [reply shared/src/androidMain/kotlin/ru/alex009/redwoodapp/navigation/FlatNavigation.kt]: fixed

**Review by Alex009 — APPROVED** [review]

**Pull request merged**